### DISCUSSION TITLE
Audio on Chrome Browser

### DISCUSSION THREAD

**Discussion by SBCounty79** [ruffle-rs/ruffle#3316] [Q&A]
I have the Ruffle extension loaded in Chrome Browser.  The site loads connect.dot connectsolutions.com the Adobe Presenter starts and the video plays without sound.  It states NO AUDIO.  In Chrome set the site to Allow Sound and MIDI and page is reloaded still without sound.  This is better than last week, the videos didn't even play.  Hoping someone and solve.  Thank you.

**Comment by flashliberty5467** [comment]
i recommend filing an issue bug what version of ruffle you are using what OS are you using windows mac etc


https://github.com/ruffle-rs/ruffle/issues/new?assignees=&labels=bug&template=bug_report.md&title=

you are likely going to either use something like cheerpx for flash kasm web browser or puffin browser from cloud mosa
